### PR TITLE
Czech language improvements

### PR DESCRIPTION
**Pull request by cewbdex**
I have done a little bit more work, also imported new strings from English file. Feel free to pull before next release. I have no work schedule and I do it at random intervals.

### CODE REVIEW

File: Mods/vcmi/config/vcmi/czech.json [hunk 1]
```diff
@@ -142,7 +174,16 @@
 	"vcmi.battleWindow.damageEstimation.kills" : "%d zahyne",
 	"vcmi.battleWindow.damageEstimation.kills.1" : "%d zahyne",
 
-	"vcmi.battleResultsWindow.applyResultsLabel" : "Použít výsledek bitvy",
+	"vcmi.battleResultsWindow.applyResultsLabel" : "Použít výsledek bitv"object.core.garrisonVertical.name"y",
```
Comment (Alexander-Wilms) [inline Mods/vcmi/config/vcmi/czech.json:177]: This is invalid JSON

Comment (cewbdex) [reply Mods/vcmi/config/vcmi/czech.json:177]: Oh thanks, don't know how this happened.

**Comment by Laserlicht** [comment]
Maybe target "beta" branch? Would be good for 1.4.2. :)

**Comment by cewbdex** [comment]
Ok, but I hate this rebasing harakiri (:
Didn't know you plan another minor release.

**Comment by cewbdex** [comment]
It worked well this time, good.

**Comment by cewbdex** [comment]
Is there any way to translate storefronts metadata?

**Comment by IvanSavenko** [comment]
storefronts, like Google Play? I think this is the only one in which we have presence.

There are some translatable strings in our Android launcher BTW: https://github.com/vcmi/vcmi/blob/develop/android/vcmi-app/src/main/res/values/strings.xml

Google Play texts are not in repo, but if you can provide translation I can publish it:
```
VCMI for Android
```
(30 characters max)
```
VCMI is an open source rewrite of the Heroes of Might and Magic 3 engine.
```
(80 characters max)
```
<b>TO USE THIS APPLICATION YOU'LL NEED ORIGINAL HEROES 3 SoD/COMPLETE GAME FILES, THEY ARE NOT PROVIDED, IT'S RECOMMENDED THAT YOU PURCHASE IT FROM GOG.com. HEROES 3 "HD EDITION" FILES ARE NOT SUPPORTED!!!</b>

Main VCMI website: https://vcmi.eu
Frequently Asked Questions: https://vcmi.eu/faq/
VCMI Wiki: https://wiki.vcmi.eu/index.php?title=Main_Page
Report bugs here: https://github.com/vcmi/vcmi/issues/
VCMI mods in latest version should be available in VCMI launcher
If you want to contact developers directly or help VCMI grow, visit us on Discord: https://discord.gg/chBT42V
=======================

VCMI is an open source engine with the purpose to rewrite the entirety of Heroes of Might and Magic 3 (optionally with WoG) from scratch, giving it new and extended possibilities. In short, it is Heroes 3 remake from scratch.

It requires the original game files to run, see https://www.youtube.com/watch?v=zGO1I3asquY for more information on how to install it (skip the first video part with installing VCMI itself from external source).

VCMI is still in development. Expect crashes and bugs (on a bright side it seems to run a lot better than the old port that was available before)
```
(4000 characters max)

**Comment by Alexander-Wilms** [comment]
> Is there any way to translate storefronts metadata?

For Linux software centers, the file https://github.com/vcmi/vcmi/blob/develop/launcher/eu.vcmi.VCMI.metainfo.xml can be translated.

Either by editing the text file directly or using [jdAppStreamEdit](https://flathub.org/apps/page.codeberg.JakobDev.jdAppStreamEdit)

```xml
[...]
<description>
    <p>VCMI is an open-source engine for Heroes of Might and Magic III with new possibilities. Years of intensive work resulted in an impressive amount of features. Among the current features are:</p>
    <p xml:lang="de">VCMI ist eine Open Source Engine für Heroes of Might and Magic III mit neuen Möglichkeiten. Jahre intensiver Arbeit haben eine beeindruckende Menge an Features ergeben. Dies sind unter anderem:</p>
[...]
```

image

**Review by IvanSavenko — APPROVED** [review]

**Pull request merged**